django/db/migrations/questioner.py

```diff
@@ -144,13 +145,13 @@ def ask_not_null_addition(self, field_name, model_name):
         """Adding a NOT NULL field to a model."""
         if not self.dry_run:
             choice = self._choice_input(
-                "You are trying to add a non-nullable field '%s' to %s without a default; "
-                "we can't do that (the database needs something to populate existing rows).\n"
+                "It is impossible to add a non-nullable field '%s' to %s without specifying a default; "
+                " because the database needs something to populate existing rows.\n"
```

Comment:
here in `ask_not_null_addition ` the text is `[...] without specifying a default; because the [...]` but in `ask_not_null_alteration` it is `[...] without providing a default because the [...]` without the semicolon. Both should probably be the same, and personally something  _feels_ off about having a because preceeded by a semicolon. (though I can't codify the rule as to _why_ in my head), YMMV.

Reply:
+1 no punctuation before because

Reply:
Typo, sorry
Thanks for catching this error

Reply:
This sentence is too long, IMO. Maybe:
```python
            choice = self._choice_input(
                f"It is impossible to add a non-nullable field '{field_name}' "
                f"to {model_name} without specifying a default. This is "
                f"because the database needs something to populate existing "
                f"rows.\n"
                f"Please select a fix:",
                [
                ...
```